src/components/Tables/MeetingVotesTable/MeetingVotesTable.stories.mdx

```diff
@@ -0,0 +1,356 @@
+import { action } from "@storybook/addon-actions";
+import { text, boolean, object } from "@storybook/addon-knobs";
+import { Meta, Props, Story } from "@storybook/addon-docs/blocks";
+import MeetingVotesTable from "./MeetingVotesTable";
+
+<Meta title="Library/Tables/MeetingVotesTable" component={MeetingVotesTable} />
+
+### An Empty Table
+
+<Props of={MeetingVotesTable} />
+
+<Story name="Empty Table">
+  <MeetingVotesTable
+    votesPage={[]}
+  />
+</Story>
+
+### A Meeting Votes Table with a Single Row
+<Story name="Single Vote">
+  <MeetingVotesTable
+    votesPage={
+    [
+      {
+        matter:{
+          id: 'example-matter-id',
+          description: "A description of this meeting",
+          name: text('example-matter-name', "Council Community Meeting"),
```

Comment:
Could you change any mentions of  `meeting` (in all these new Meeting Votes Table stories) to `legislation`?

Reply:
Strong agree. and just make up matter names:

"CB1234", "Res 5678", etc.